Dockerfile

```diff
@@ -14,8 +14,16 @@ ENV PYTHON_VERSION_36 3.6.8
 ENV PYTHON_VERSION_37 3.7.3
 ENV PYTHON_VERSION_38 3.8.0
 ENV PYPY_VERSION_35 pypy3.5-7.0.0
+
 # Note: 4.7.12.1 drastically increases memory usage
 ENV CONDA_VERSION 4.6.14
+
+# Use _PIP_VERSION (with starting _) because of a bug in Pip
+# https://github.com/pypa/pip/issues/4528
+ENV _PIP_VERSION 20.0.1
+
+ENV SETUPTOOLS_VERSION 45.1.0
+ENV VIRTUALENV_VERSION 16.7.9
```

Comment:
I'd like these to go _after_ the pyenv install, so that we don't have to rebuild all python versions to change pip versions.

Reply:
That can be another PR tho, so we don't block this one.